src/components/utils/popover/popover-inline.ts

```diff
@@ -0,0 +1,185 @@
+import { isMobileScreen } from '../../utils';
+import { PopoverItem, PopoverItemDefault, PopoverItemType, WithChildren } from './components/popover-item';
+import { PopoverItemHtml } from './components/popover-item/popover-item-html/popover-item-html';
+import { PopoverDesktop } from './popover-desktop';
+import { CSSVariables, css } from './popover.const';
+import { PopoverParams } from './popover.types';
+
+/**
+ * Horizontal popover that is displayed inline with the content
+ */
+export class PopoverInline extends PopoverDesktop {
+  /**
+   * Item nested popover is displayed for
+   */
+  private nestedPopoverTriggerItem: PopoverItemDefault | PopoverItemHtml | null = null;
+
+  /**
+   * Constructs the instance
+   *
+   * @param params - instance parameters
+   */
+  constructor(params: PopoverParams) {
+    const isHintEnabled = !isMobileScreen();
+
+    super(
+      {
+        ...params,
+        class: css.popoverInline,
+      },
+      {
+        [PopoverItemType.Default]: {
+          /**
+           * We use button instead of div here to fix bug associated with focus loss (which leads to selection change) on click in safari
+           *
+           * @todo figure out better way to solve the issue
+           */
+          wrapperTag: 'button',
+          hint: {
+            position: 'top',
+            alignment: 'center',
+            enabled: isHintEnabled,
+          },
+        },
+        [PopoverItemType.Html]: {
+          hint: {
+            position: 'top',
+            alignment: 'center',
+            enabled: isHintEnabled,
+          },
+        },
+      }
+    );
+
+    /**
+     * If active popover item has children, show them.
+     * This is needed to display link url text (which is displayed as a nested popover content)
+     * once you select <a> tag content in text
+     */
+    this.items
+      .forEach((item) => {
+        if (!(item instanceof PopoverItemDefault) && !(item instanceof PopoverItemHtml)) {
+          return;
+        }
+
+        if (item.hasChildren && item.isChildrenExpanded) {
+          this.showNestedItems(item);
+        }
+      });
+  }
+
+  /**
+   * Returns visible element offset top
+   */
+  public get offsetLeft(): number {
+    if (this.nodes.popoverContainer === null) {
+      return 0;
+    }
+
+    return this.nodes.popoverContainer.offsetLeft;
+  }
+
+  /**
+   * Open popover
+   */
+  public override show(): void {
+    /**
+     * If this is not a nested popover, set CSS variable with width of the popover
+     */
+    if (this.nestingLevel === 0) {
+      this.nodes.popover.style.setProperty(
+        CSSVariables.InlinePopoverWidth,
+        this.size.width + 'px'
+      );
+    }
+    super.show();
+  }
+
+  /**
+   * Disable hover event handling
+   */
+  protected override handleHover(): void {
+    // Do nothing
```

Comment:
maybe `return`? Right now it looks like this method is redundant, but it is needed to override parent class behavior, as is understood. 
```suggestion
    return;
```

Reply:
fixed